astropy/units/core.py

```diff
@@ -1140,6 +1140,8 @@ def compose(self, equivalencies=[], units=None, max_depth=2,
             composite units.
 
         include_prefix_units : bool, optional
+            Default is `True` if a sequence is passed in to ``units``,
+            `False` otherwise.
             When `True`, include prefixed units in the result.
```

Comment:
THis got a bit mangled. Could you change it to
```
When `True`, include prefixed units in the result.
Default is `True` if a sequence is passed in to ``units``,
`False` otherwise.
```